src/browser/modules/Stream/FrameContainer.tsx

```diff
@@ -0,0 +1,166 @@
+import ParamsFrame from './ParamsFrame'
+import ErrorFrame from './ErrorFrame'
+import HelpFrame from './HelpFrame'
+import CypherScriptFrame from './CypherScriptFrame/CypherScriptFrame'
+import SchemaFrame from './SchemaFrame'
+import StyleFrame from './StyleFrame'
+import SysInfoFrame from './SysInfoFrame/SysInfoFrame'
+import ConnectionFrame from './Auth/ConnectionFrame'
+import DisconnectFrame from './Auth/DisconnectFrame'
+import ServerStatusFrame from './Auth/ServerStatusFrame'
+import ServerSwitchFrame from './Auth/ServerSwitchFrame'
+import UseDbFrame from './Auth/UseDbFrame'
+import ChangePasswordFrame from './Auth/ChangePasswordFrame'
+import QueriesFrame from './Queries/QueriesFrame'
+import UserList from '../User/UserList'
+import UserAdd from '../User/UserAdd'
+import DbsFrame from './Auth/DbsFrame'
+import { ExportItem } from '../Frame/ExportButton'
+
+const nameToFrame: Record<string, React.ComponentType<any>> = {
+  error: ErrorFrame,
+  cypher: CypherFrame,
+  'cypher-script': CypherScriptFrame,
+  'user-list': UserList,
+  'user-add': UserAdd,
+  'change-password': ChangePasswordFrame,
+  pre: PreFrame,
+  play: PlayFrame,
+  'play-remote': PlayFrame,
+  history: HistoryFrame,
+  param: ParamsFrame,
+  params: ParamsFrame,
+  connection: ConnectionFrame,
+  disconnect: DisconnectFrame,
+  schema: SchemaFrame,
+  help: HelpFrame,
+  queries: QueriesFrame,
+  sysinfo: SysInfoFrame,
+  status: ServerStatusFrame,
+  'switch-success': ServerSwitchFrame,
+  'switch-fail': ServerSwitchFrame,
+  'use-db': UseDbFrame,
+  'reset-db': UseDbFrame,
+  dbs: DbsFrame,
+  style: StyleFrame,
+  default: DefaultFrame
+}
+
+const getFrameComponent = (frameData: FrameStack): React.ComponentType<any> => {
+  const { cmd, type } = frameData.stack[0]
+  let MyFrame = nameToFrame[type] || nameToFrame.default
+
+  if (type === 'error') {
+    try {
+      const command = cmd.replace(/^:/, '')
+      const Frame = command[0].toUpperCase() + command.slice(1) + 'Frame'
+      MyFrame = require('./Extras/index')[Frame] || nameToFrame['error']
+    } catch (e) {}
+  }
+  return MyFrame
+}
+
+type FrameContainerProps = {
+  frameData: FrameStack
+  activeConnectionData: Connection | null
+}
+
+export function FrameContainer(props: FrameContainerProps): JSX.Element {
+  const {
+    isFullscreen,
+    toggleFullscreen,
+    isCollapsed,
+    toggleCollapse
+  } = useSizeToggles()
+  const frame = props.frameData.stack[0]
+  const [exportItems, setExportItems] = useState<ExportItem[]>([])
+  const frameProps: BaseFrameProps = {
+    frame: { ...frame, isPinned: props.frameData.isPinned },
+    activeConnectionData: props.activeConnectionData,
+    stack: props.frameData.stack,
+    isFullscreen,
+    isCollapsed,
+    setExportItems
+  }
+  const FrameComponent = getFrameComponent(props.frameData)
+
+  return (
+    <StyledFrame
+      className={isFullscreen ? 'is-fullscreen' : ''}
+      data-testid="frame"
+      fullscreen={isFullscreen}
```

Comment:
```suggestion
      ifFullscreen={isFullscreen}
```

Reply:
I guess you forgot to make changes here?

Reply:
Ah missed it in my search and replace.. thanks!

Reply:
Got a few extra unrelated ones now as well, to make the whole code base consistent